pyment/pymentapp.py

```diff
@@ -138,6 +149,9 @@ def main():
     source = args.path
 
     files = get_files_from_dir(source)
+    if not files:
+        msg = BaseException("No files were found matching %s" % args.path)
```

Comment:
Minor nitpick: The rest of the codebase seems to use the `.format` style, so this line may be better off as:

`msg = BaseException("No files were found matching {0}".format(args.path))`

Reply:
I've chased down the other % formatting and changed them to format.

Fortunately pycharm change change it to format automatically since I never use it.